lang/en/docs/cli/unlink.md

```diff
@@ -4,9 +4,11 @@ guide: docs_cli
 layout: guide
 ---
 
+{% include vars.html %}
```

Comment:
You are right, it is. I basically just copied over the same requirements from the other md files in the cli folder (which all contain this `include`). I can remove it if you want.